lib/Basic/LangOptions.cpp

```diff
@@ -356,6 +364,25 @@ std::pair<bool, bool> LangOptions::setTarget(llvm::Triple triple) {
     break;
   }
 
+  // Set the "_native_word_size" platform condition.
```

Comment:
Nit:
```suggestion
  // Set the "_pointerBitWidth" platform condition.
```

But also could you document where these particular targets come from so we can keep things in sync in the future?